sds/service.go

```diff
@@ -84,15 +86,19 @@ func (srv *Service) Stop() error {
 
 // Register registers the sds.Service into the given gRPC server.
 func (srv *Service) Register(s *grpc.Server) {
-	discovery.RegisterSecretDiscoveryServiceServer(s, srv)
+	secret.RegisterSecretDiscoveryServiceServer(s, srv)
+}
+
+func (srv *Service) DeltaSecrets(sds secret.SecretDiscoveryService_DeltaSecretsServer) (err error) {
+	return errors.New("DeltaSecretsServer not implemented")
```

Comment:
This works if the `api_type` is set to `GRPC`. See https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/config_source.proto.html?highlight=api_type#envoy-v3-api-enum-config-core-v3-apiconfigsource-apitype.